package.json

```diff
@@ -6,6 +6,7 @@
     "lint": "node fern.js lint",
     "build": "node fern.js build configs/sandbox.js --environment prod",
     "build-module": "node fern.js build configs/module.js --environment prod",
+    "postinstall": "patch-package",
```

Comment:
Not sure here: will this be called on an external installation? Should `patch-package` be in non-dev dependencies? @remusao

Reply:
As long as this works when included into `brave-core` I think it's fine (the dev dependencies should work there but we'll need to double-check). IMO it's still a dev dependency if it's install/build-time vs. `dependencies` (which I see a runtime dependencies bundled with the extension).